source/tutorial/acceptance-test.md

```diff
@@ -19,7 +19,7 @@ At this point, you can continue to the [next page](../routes-and-templates/) or
 
 ### Testing Our Application As We Go
 
-We can represent the goals above as [Ember acceptance tests](../../testing/acceptance/). Acceptance tests interact with our app like an actual person would, but are automated, helping ensure that our app doesn't break in the future.
+We can represent the goals above as [Ember acceptance tests](../../testing/acceptance/). Application tests interact with our app like an actual person would, but are automated, helping ensure that our app doesn't break as you make changes.
```

Comment:
So `application` tests?  Is this a new term we're introducing? If so how do we announce this to the community?

Reply:
yes, with the new testing APIs "acceptance" tests are replaced by "application" tests.